osschallenge/views.py

```diff
@@ -150,35 +149,26 @@ def shorten(string, max_length):
 
 
 def TaskIndexView(request):
```

Comment:
`TaskIndexView` and `MyTaskIndexView` look very similar. Can you merge them into one view?

Reply:
It's finally done :smiley: